ios/ReactNativeCharts/RNCharts-Bridging-Header.h

```diff
@@ -0,0 +1,7 @@
+#import "React/RCTBridge.h"
+#import "React/RCTViewManager.h"
+#import "React/RCTUIManager.h"
+#import "React/UIView+React.h"
+#import "React/RCTBridgeModule.h"
+#import "React/RCTEventDispatcher.h"
+#import "React/RCTEventEmitter.h"
```

Comment:
missing

    #import "React/RCTFont.h"

Reply:
that is no need here